### DISCUSSION TITLE
Formatting of Windows file paths in YAML

### DISCUSSION THREAD

**Discussion by rodti** [swar/Swar-Chia-Plot-Manager#681] [Q&A]
I've read through a number of issues and discussions here (as well as some Googling) and just can't seem to format my Windows file paths properly in the config file. I have two temp drives (D: and E:) and one destination drive (Z:). For all three I use the root folder for my temp and plot files.

I've tried formatting the temporary directory as both a single or multiple value (observing the two space, dash, space formatting rule):

```
    temporary_directory:  D:\

    temporary_directory:
      - D:\
      - E:\
```

I've tried formatting the file paths using the following formats:

```
    D:
    D:\
    'D:\'
    "D:\"
    "D:\\"
```

And nothing seems to work. The debug log always suggests an issue with this section of the config. I've tried YAMLLint but its suggestions didn't work.

I've spent long enough trying to work out for myself that it's time to ask for help! Can anyone confirm a Windows file format for drive roots that is known to work?

Many thanks in advance :-)

**Comment by rodti** [comment]
Ran using specifically python3 and now it works. How strange! For clarity, it works in both single and lists, and with D:\ style formatting.

**Comment by Icabaad** [comment]
You have probably covered this with YAMLLint but remember not to use tabs in a YAML file. has to be spaces.

**Reply by rodti** [reply]
Definitely spaces only! Seems to be something peculiar to running Python on Windows 10. I'd installed Python 3.9 from the Windows Store. In Powershell 'python ./manager start' doesn't work but 'python3 ./manager start' does!
